src/plugins/telemetry/server/telemetry_collection/get_local_license.ts

```diff
@@ -17,60 +17,61 @@
  * under the License.
  */
 
-import { LegacyAPICaller } from 'kibana/server';
 import { ESLicense, LicenseGetter } from 'src/plugins/telemetry_collection_manager/server';
+import { ElasticsearchClient } from '../../../../../src/core/server';
 
 let cachedLicense: ESLicense | undefined;
 
-function fetchLicense(callCluster: LegacyAPICaller, local: boolean) {
-  return callCluster<{ license: ESLicense }>('transport.request', {
-    method: 'GET',
-    path: '/_license',
-    query: {
-      local,
-      // For versions >= 7.6 and < 8.0, this flag is needed otherwise 'platinum' is returned for 'enterprise' license.
-      accept_enterprise: 'true',
-    },
-  });
-}
```

Comment:
I changed the internals because the shape of the response has [changed](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION_EXAMPLES.md#elasticsearch-client). `callCluster` either returns a response (successful call) or an error. The new client returns a response body _regardless_ of returning an error. This might also be what's causing the issues with the failed unauthorized `api/stats` call? What do you think?